zpeclient.go

```diff
@@ -410,3 +412,17 @@ func (c *Cache) updateCacheStatus(timestamp string) error {
 	c.cacheStatus = CacheStale
 	return nil
 }
+
+// setCacheEnabled - sets that the cache is enabled
+func (c *Cache) SetCacheEnabled() {
```

Comment:
Could we combine these two functions into one and pass in the bool value it should be set to as a parameter?